posydon/utils/x_ray_luminosities.py

```diff
@@ -0,0 +1,188 @@
+    of the accretion disc that beams the outgoing X-ray emission. This is
+    important for observing super-Eddington X-ray sources
+    (e.g. ultraluminous X-ray sources). In the case of a BH we are assuming that it
+    has zero spin which is not a good approximation for high accretion rates.
+    Parameters
+    ----------
+    binary : BinaryStar
+        The binary object.
+    Returns
+    -------
+    list
+        The super-Eddington isotropic-equivalent accretion rate and beaming
+        factor respcetively in solar units.
+    References
+    ----------
+    .. [1] Shakura, N. I. & Sunyaev, R. A. 1973, A&A, 24, 337
+    .. [2] King A. R., 2008, MNRAS, 385, L113
+    """
+    if binary.star_1.state in ['NS', 'BH']:
+        accretor = binary.star_1
+        donor = binary.star_2
+        don_rel_RL = binary.rl_relative_overflow_2
+    elif binary.star_2.state in ['NS', 'BH']:
+        accretor = binary.star_2
+        donor = binary.star_1
+        don_rel_RL = binary.rl_relative_overflow_1
+    else:
+        return 0.0, 1.0
+
+    if ((binary.lg_mtransfer_rate is None) and (accretor.lg_mdot is None)):
+        return 0.0, 1.0
+    elif (binary.lg_mtransfer_rate is not None):
+        mdot = 10**binary.lg_mtransfer_rate
+    else:
+        mdot = 10**accretor.lg_mdot
+
+    if don_rel_RL is None:
+        return 0.0, 1.0
+
+    # Mdot_edd converted from solar to cgs units
+    mdot_edd = eddington_limit(binary, idx=-1)[0] * (const.msol / const.secyer)
+    eta = eddington_limit(binary, idx=-1)[1]
+    # Donor Roche-lobe radius don_RL
+    don_RL = 10**donor.log_R / (don_rel_RL + 1)
+
+    if ( (donor.state == 'H-rich_Core_H_burning') & (donor.mass >= 3.0) &
+        (binary.state == 'detached') & (don_RL <= (100.0 * 10 ** donor.log_R) ) & 
+        (donor.surf_avg_omega_div_omega_crit >= 0.7) & (binary.orbital_period >= 10.0) & (binary.orbital_period <= 300.0) ): 
+        # Be-XRBs
+        Lx = 10.0**35 * 10.0**(4.53 - (1.50 * np.log10(binary.orbital_period)) )
+        b = 0.1 # assuming 10% duty cycle
+    else: 
+        # RLO and wind
+        if mdot >= mdot_edd:
+            if (mdot > 150.0 * mdot_edd):
+                b = 3.2e-3
+            elif (mdot > 8.5 * mdot_edd):
+                b = 73. / (mdot / mdot_edd)**2
+            else:
+                b = 1.
+            Lx = mdot_edd*eta * const.clight**2*(1 + np.log((10**binary.lg_mtransfer_rate) / mdot_edd))/b
+        else:
+            b = 1.
+            Lx = 10**accretor.lg_mdot*eta*const.clight**2
+
+    # If observe_wind_XRB == True, check if wind accretion rate is high enough to make accretion disc
+    # bychecking the donor is filling 80% of its Roche lobe.
+    # Hirai & Mandel (2021)
+    if (observe_wind_XRB == True) & ((state == 'detached') & (acc_state == 'BH')):
+        if (( don_rel_RL + 1 ) < 0.8):
+            Lx = 10**-54
+            b = 1.0
+        
+    return Lx, b
+
+    
+    
+
+
+# def beaming(binary):
```

Comment:
Please explain, why here is code, which isn't aimed to be used. May consider to simply remove it.